opta/commands/generate_terraform.py

```diff
@@ -98,9 +98,12 @@ def generate_terraform(
     config = check_opta_file_exists(config)
 
     pre_check()
+    input_variables = InputVariable.parse_string_to_dict(var)
```

Comment:
can you check if we can use this [callback](https://click.palletsprojects.com/en/8.0.x/options/#callbacks-for-validation) instead of having to do this call in every command?

```python
input_variable_option = click.option(
    "--var",
    "-v",
    multiple=True,
    default=[],
    action=;
    help="Add input variables to your yaml at run time (e.g. --var variable1=value1)",
    callback=parse_string_to_dict #would be called automatically, would need to be changed to work with api
)
```

Reply:
done